### PR TITLE
Add more Gauntlet sets + misc. fixes

### PR DESCRIPTION
**Pull request by FrostedGeulleisia**
idk if renaming the file may break something but eh

### CODE REVIEW

**Comment by FrostedGeulleisia** [comment]
Update: some more changes
Detailed changelog below.

Sets are taken from each run's "Final" save, Gauntlet X not present due to me not having that save yet (also, the only non-trade mon that's compatible is Moth). Present are only Generation 1-4 species, and only ones that either have entered the Hall of Fame, or are iconic enough (Platinum Palkia/Arceus may be controversial, but I've also included them in the rematches in Gauntlet X).
The sets are unaltered aside from things listed below, which means legendaries aren't nerfed or anything. Feel free to suggest changes if you feel they have to be nerfed.
Changes from original sets:
- Mandatory conversion (might not be 100% accurate) for Gauntlet Crystal sets.
- Made main Gauntlet Emerald sets hold Heart Mails instead of Harbor/Orange/Wave Mails due to those not existing in Gen 4,
- Removed Griseous Orb from AK (Giratina) since it's a Platinum-only item.
- Changed some names due to incompatibilities, mainly Aggron-TPPGauntletEmerald, Leafeon-TPPGauntletPlatinum and Darkrai-TPPGauntletBlack2
- Made APSEPIEMG (Porygon-Z)'s ability Download, since Analtyic is a Generation 5 ability (Download was the line's signature ability until BW, where Genesect gained it too).
Note: AACm   neg (Lumineon) has 18 Surf PP, due to a PP Up being applied to it at some point.

Unrelated changes:
- Fixed filenmes of some shiny sets, specifically Glaceon-Shiny (472_Glaceon.yaml > 471_Glaceon.yaml) and Drifblim-Shiny (426_Driflblim.yml > 426_Driflblim.yaml), this's purely a cosmetic change.

File: Runmon/43_GauntletEmerald.yaml [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    - [Focus Energy]
+    - [Ember]
+    - [Fly]
+happiness: 237
+biddable: True
+rarity: 1.0
+shiny: False
+nature: Careful
+ivs: {hp: 1, atk: 17, def: 4, spA: 12, spD: 20, spe: 9}
+evs: {hp: 82, atk: 114, def: 31, spA: 17, spD: 62, spe: 20}
```
Comment (mathfreak231) [inline Runmon/43_GauntletEmerald.yaml:147]: Try to track down MMDfll the Whiscash, too.

File: Runmon/44_GauntletPlatinum.yaml [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+gender: [m]
+setname: TPP Gauntlet Platinum
+tags: [runmon, in-game]
+item: [Razor Claw]
+ability: [Overgrow]
+moves:
+    - [Razor Leaf]
+    - [Curse]
+    - [Synthesis]
+    - [Rock Climb]
```
Comment (mathfreak231) [inline Runmon/44_GauntletPlatinum.yaml:123]: BFG had Earthquake for both Cynthia 1 (didn't have Synthesis) and Cynthia 2 (didn't have Rock Climb). I'm tempted to add EQ as an alternative in both of those move slots.

File: Runmon/45_GauntletBlazeBlack2.yaml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+ivs: {hp: 0, atk: 30, def: 25, spA: 30, spD: 7, spe: 30}
+evs: {hp: 56, atk: 123, def: 97, spA: 102, spD: 67, spe: 65}
+---
+species: Porygon-Z
+ingamename: "APSEPIEMG"
+displayname: "APSEPIEMG"
+setname: TPP Gauntlet Black 2
+tags: [runmon, in-game]
+item: [Wise Glasses]
+ability: [Download]
```
Comment (mathfreak231) [inline Runmon/45_GauntletBlazeBlack2.yaml:29]: According to VODs, its ability is Analytic.

File: Runmon/45_GauntletBlazeBlack2.yaml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    - [Dark Void]
+    - [Nasty Plot]
+happiness: 233
+biddable: True
+rarity: 1.0
+ball: [Poké]
+shiny: False
+nature: Bashful
+ivs: {hp: 2, atk: 25, def: 24, spA: 12, spD: 11, spe: 27}
+evs: {hp: 64, atk: 137, def: 60, spA: 119, spD: 63, spe: 67}
```
Comment (mathfreak231) [inline Runmon/45_GauntletBlazeBlack2.yaml:63]: Try to track down lier0 6 the Dusknoir as well.

File: Runmon/45_GauntletBlazeBlack2.yaml (hunk 3 above)
Comment (mathfreak231) [inline Runmon/45_GauntletBlazeBlack2.yaml:29]: I think I would prefer Adaptabiliy/Download alternation. Seems better than arbitrarily picking one.

File: Runmon/44_GauntletPlatinum.yaml (hunk 2 above)
Comment (mathfreak231) [reply Runmon/44_GauntletPlatinum.yaml:123]: Others have suggested replacing Rock Climb with EQ entirely. I'll let this one sit while the missing mons I mentioned are being searched for.

**Comment by FrostedGeulleisia** [comment]
Not entirely sure how pbrEngine will handle Earthquake being an alternate move in two different slots for BFG, might just remove it from the Synthesis slot if it poses problems (compiling reported no warnings here).

File: Runmon/44_GauntletPlatinum.yaml [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+gender: [m]
+setname: TPP Gauntlet Platinum
+tags: [runmon, in-game]
+item: [Razor Claw]
+ability: [Overgrow]
+moves:
+    - [Razor Leaf]
+    - [Curse]
+    - [Synthesis, Earthquake]
+    - [Rock Climb, Earthquake]
```
Comment (mathfreak231) [inline Runmon/44_GauntletPlatinum.yaml:123]:
```suggestion
    - [Rock Climb, Earthquake]
separations:
    - [Earthquake, Earthquake]
```
Have to do this so it doesn't end up with EQ in both slots.

**Comment by mathfreak231** [comment]
Thank you!

**Pull request merged**